QuickFixers/QuickFixers.csproj

```diff
@@ -243,9 +245,14 @@
     <Content Include="Views\ServiceProvider\ServicesOffered.cshtml" />
     <Content Include="Views\ServiceProvider\ScheduledServices.cshtml" />
     <Content Include="Views\ServiceProvider\PastServices.cshtml" />
+    <Content Include="Views\Home\MainPage.cshtml" />
```

Comment:
Can you do a Clean and Rebuild? This might be a page we removed before the Home page merged in.